src/index.js

```diff
@@ -20,23 +21,40 @@ import output from './output';
  * This is adapter function that will send the data to the chosen output module
  * @param {*} data 
  */
-const sendToOutput = (...data) => {
+const sendToOutput = (data) => {
   output(data);
 };
 
 /**
  * This function will decode the transaction and the logs that happed inside it,
```

Comment:
Decode a transaction and all logs generated from it then send results to output model

Reply:
👍